### PR TITLE
Add documentation section about Cloudflare ES6 Module Worker

### PR DESCRIPTION
**Pull request by G4brym**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ In the example below, the `ToDoList` route will have an Integer parameter called
 
 Then the page number will be available inside the `handle()` function in the data object passed in the argument.
 
+Take notice that the `data` object is always the **last argument** the `handle()` function receives.
```
Comment (celso) [inline README.md:49]: that the handle()

File: README.md [hunk 2]
```diff
@@ -223,6 +225,8 @@ This is where you will use the Schema types explained above.
 
 Example path parameter:
 
+*Notice the parameter key needs to be the same name as in the route path*
```
Comment (celso) [inline README.md:228]: - Notice that
- as the route path

File: README.md [hunk 3]
```diff
@@ -275,7 +279,73 @@ router.get('/todos', ToDoList)
 
 ## Advanced Usage
 
-### 1. Using Typescript types
+### 1. Cloudflare ES6 Module Worker
```
Comment (celso) [inline README.md:282]: - argument is split
- that the handle()

**Review by celso — APPROVED** [review]

**Comment by blackbee-cloud** [comment]
f5192ef91614fb883e85f2f30c69d7a730416ba0

**Pull request merged**